orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/model/Execution.java

```diff
@@ -53,6 +60,7 @@
   AuthenticationDetails authentication;
   PausedDetails paused;
   ExecutionEngine executionEngine = DEFAULT_EXECUTION_ENGINE;
+  String origin;
```

Comment:
Anything needed to ensure this gets persisted correctly? Don't recall if we need to do this in the `ExecutionRepository` for first level attributes.